### ISSUE TITLE
download_clinical throws error

### ISSUE THREAD

**Issue by shucon** [Christensen-Lab-Dartmouth/PyMethylProcess#11]
I'm trying to download the clinical information on a Docker Machine using `pymethyl-preprocess download_clinical` and I'm getting the following error:

![Screenshot from 2020-11-01 18-53-41](https://user-images.githubusercontent.com/22860775/97804090-9f0a4800-1c73-11eb-8f9b-6188d9859157.png)

PS: It would be great if someone can upload the clinical data for TCGA. So that I can continue working in the meantime. :smiley:

**Comment by jlevy44** [comment]
[clinical_info.csv.zip](https://github.com/Christensen-Lab-Dartmouth/PyMethylProcess/files/5488857/clinical_info.csv.zip)
It's been a while, but let me know if this is what you're looking for!

**Comment by jlevy44** [comment]
I'm assuming that you're following this?: 
https://github.com/Christensen-Lab-Dartmouth/PyMethylProcess/blob/master/example_scripts/TCGA.md

**Comment by shucon** [comment]
> I'm assuming that you're following this?:
> https://github.com/Christensen-Lab-Dartmouth/PyMethylProcess/blob/master/example_scripts/TCGA.md

@jlevy44 Yes, I'm following that. Although, now I have the clinical info I was able to continue with the rest of the pipeline without any major hiccups.

So if it's not a priority issue, you can keep it on hold.

Thanks a lot. :smiley:

**Comment by jlevy44** [comment]
Thanks! Sorry this function seems to have been problematic as of late. Definitely on my to-do list!
